8353671: Remove dead code missed in JDK-8350459

vpaprotsk opened the pull request:
<!--
Replace this text with a description of your pull request (also remove the surrounding HTML comment markers).
If in doubt, feel free to delete everything in this edit box first, the bot will restore the progress section as needed.
-->

<!-- Anything below this marker will be automatically updated, please do not edit manually! -->
---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8353671](https://bugs.openjdk.org/browse/JDK-8353671): Remove dead code missed in JDK-8350459 (**Sub-task** - P4)


### Reviewers
 * [Sandhya Viswanathan](https://openjdk.org/census#sviswanathan) (@sviswa7 - **Reviewer**)
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/24423/head:pull/24423` \
`$ git checkout pull/24423`

Update a local copy of the PR: \
`$ git checkout pull/24423` \
`$ git pull https://git.openjdk.org/jdk.git pull/24423/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 24423`

View PR using the GUI difftool: \
`$ git pr show -t 24423`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/24423.diff">https://git.openjdk.org/jdk/pull/24423.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/24423#issuecomment-2776649376)
</details>

vpaprotsk commented:
@ascarpino If you wouldn't mind, should be a quick one :)

seanjmullan reviewed (approved):
Can you add a link to JDK-8350459 in the JBS issue?

Also, Tony is not available right now to review, so I reviewed and approved it.

seanjmullan commented:
Also, the JBS issue needs an appropriate `noreg` label.

vpaprotsk commented:
> Also, the JBS issue needs an appropriate `noreg` label.

Added `noreg-cleanup`, I think thats the best match (?)

> Can you add a link to JDK-8350459 in the JBS issue?

Its already a subtask of JDK-8350459, so its 'linked' in a way (though the UI really should display the parent task in the child task, just like it does in the parent task). 

I went ahead and added a 'relates-to' link anyway..

Thanks @seanjmullan  for the approval!

/integrate

sviswa7 commented:
/sponsor

seanjmullan commented:
Are you available to help with any CI issues that may occur in the next few hours if I sponsor now? If not, let me know when is a good time (with timezone).
